app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java

```diff
@@ -0,0 +1,266 @@
+
+        // Adding child data
+        List<String> assault = new ArrayList<String>();
+        assault.add(context.getString(R.string.asexual_assault));
+
+        List<String> assailant = new ArrayList<String>();
+        assailant.add(context.getString(R.string.assailant_info));
+
+        List<String> burglary = new ArrayList<String>();
+        burglary.add(context.getString(R.string.burglary_info));
+
+        List<String> intervention = new ArrayList<String>();
+        intervention.add(context.getString(R.string.intervention_info));
+
+        List<String> phenomenon = new ArrayList<String>();
+        phenomenon.add(context.getString(R.string.phenomenon_info));
+
+        List<String> cyber = new ArrayList<String>();
+        cyber.add(context.getString(R.string.cyber_info));
+
+        List<String> danger = new ArrayList<String>();
+        danger.add(context.getString(R.string.danger_info));
+
+        List<String> mitigate = new ArrayList<String>();
+        mitigate.add(context.getString(R.string.mitigation_info));
+
+        List<String> pii = new ArrayList<String>();
+        pii.add(context.getString(R.string.pii_info));
+
+        List<String> rape = new ArrayList<String>();
+        rape.add(context.getString(R.string.rape_info));
+
+        List<String> risk = new ArrayList<String>();
+        risk.add(context.getString(R.string.risk_info));
+
+        List<String> rob = new ArrayList<String>();
+        rob.add(context.getString(R.string.robbery_info));
+
+        List<String> safe = new ArrayList<String>();
+        safe.add(context.getString(R.string.safety_info));
+
+        List<String> security = new ArrayList<String>();
+        security.add(context.getString(R.string.security_info));
+
+        List<String> sexual_assault = new ArrayList<String>();
+
+        sexual_assault.add(context.getString(R.string.sexual_assault_info1));
+
+        List<String> exploit = new ArrayList<String>();
+        exploit.add(context.getString(R.string.exploitation_info));
+
+        List<String> harass = new ArrayList<String>();
+        harass.add(context.getString(R.string.harassment_info));
+
+        List<String> misconduct = new ArrayList<String>();
+        misconduct.add(context.getString(R.string.misconduct_info));
+
+        List<String> threat = new ArrayList<String>();
+        threat.add(context.getString(R.string.threat_info));
+
+        List<String> predator = new ArrayList<String>();
+        predator.add(context.getString(R.string.predator_info));
+
+        List<String> stalk = new ArrayList<String>();
+        stalk.add(context.getString(R.string.stalking_info));
+
+        List<String> theft = new ArrayList<String>();
+        theft.add(context.getString(R.string.theft_info));
+
+        List<String> vulnerability = new ArrayList<String>();
+        vulnerability.add(context.getString(R.string.vulnerability_info));
+
+        listDataChild.put(listDataHeader.get(0), assault);
```

Comment:
Try to add these to an resource array and iterating rather than doing this one by one. The previous approach I suggested may help.